prody/dynamics/plotting.py

```diff
@@ -211,7 +211,7 @@ def showProjection(ensemble, modes, *args, **kwargs):
       * 3 modes: :meth:`~mpl_toolkits.mplot3d.Axes3D.plot`"""
 
     import matplotlib.pyplot as plt
-
+    plt.figure()
```

Comment:
I can search through and work out where to put the option. If you know already then maybe it's better for you to do it.